components/newAssessmentForm.js

```diff
@@ -0,0 +1,66 @@
+import { Container, Row, Form, Col, Button } from 'react-bootstrap'
+import React, { useState } from 'react';
+import Input from './Input'
+
+export default function newDRAForm(props) {
+
+  const [job, setJob] = useState({
+    JobSite: "",
+    JobAddress: "",
+    JobSiteDescription: "",
+    Phone: "",
+    Date: "",
+    Time: ""
+  })
+
+  const onSubmit = (e) => {
+    e.preventDefault();
+    //Needed to be added to database to store
+    alert(`Submitting ${job.JobSite}, ${job.JobAddress}, ${job.JobSiteDescription}, ${job.Phone}, ${job.Date}, ${job.Time}`);
+
+  }
+  const inputsHandler = (e) => {
+    // update the attributes in object
+    const name = e.target.name;
+    const value = e.target.value;
+    job[name] = value;
+    setJob(job);
+
+  }
+
+  return (
+    <>
+      <Container>
+        <Row>
+          <h2>Create new Assessment</h2>
+        </Row>
+        <Form>
+          <label>Job Site:</label>
+          <Input type="text" placeholder="Name of Location" name="JobSite" onChange={inputsHandler} />
```

Comment:
see previous. Inputs should be inside label, otherwise React sees it as a random label belonging to nothing